e2e-tests/cypress/support/commands.ts

```diff
@@ -16,33 +20,46 @@ Cypress.Commands.add('post', ({ url, user = `${Cypress.env().login}:${Cypress.en
 
 Cypress.Commands.add(
   'import',
-  ({ url, filename, user = `${Cypress.env().login}:${Cypress.env().password}` }, ...args) =>
-    cy.exec(`curl -v -k -H "kbn-xsrf: true" --form file=@${filename}  -X POST "${url}" --user ${user}`).then(result => {
+  ({url, filename, user = `${ Cypress.env().login }:${ Cypress.env().password }`}, ...args) =>
+    cy.exec(`curl -v -k -H "kbn-xsrf: true" --form file=@${ filename }  -X POST "${ url }" --user ${ user }`).then(result => {
       console.log(url, result);
       return isJsonString(result.stdout) ? JSON.parse(result.stdout) : result.stdout;
     })
 );
 
 Cypress.Commands.add(
   'getRequest',
-  ({ url, user = `${Cypress.env().login}:${Cypress.env().password}`, header }, ...args) =>
-    cy.exec(`curl -v -k "${url}"  --user ${user}  ${header ? `-H "${header}"` : ''}`).then(result => {
+  ({url, user = `${ Cypress.env().login }:${ Cypress.env().password }`, header}, ...args) =>
+    cy.exec(`curl -v -k "${ url }"  --user ${ user }  ${ header ? `-H "${ header }"` : '' }`).then(result => {
       console.log(url, result);
       return isJsonString(result.stdout) ? JSON.parse(result.stdout) : result.stdout;
     })
 );
 
 Cypress.Commands.add(
   'deleteRequest',
-  ({ url, header, user = `${Cypress.env().login}:${Cypress.env().password}` }, ...args) =>
+  ({url, header, user = `${ Cypress.env().login }:${ Cypress.env().password }`}, ...args) =>
     cy
-      .exec(`curl  -H "kbn-xsrf: true" -v -k -X DELETE "${url}"  --user ${user} ${header ? `-H "${header}"` : ''}`)
+      .exec(`curl  -H "kbn-xsrf: true" -v -k -X DELETE "${ url }"  --user ${ user } ${ header ? `-H "${ header }"` : '' }`)
       .then(result => {
         console.log(url, result);
         return isJsonString(result.stdout) ? JSON.parse(result.stdout) : result.stdout;
       })
 );
 
+Cypress.Commands.add('verifyDownload', (fileName: string, options) => {
+  const downloadsFolder = Cypress.config().downloadsFolder;
+  const downloadTimeout = options.timeout || 60000; // Default timeout of 60 seconds
```

Comment:
Why do we need to set timeout time explicitly? Currently, we have some defaults in the [config](https://github.com/beshu-tech/readonlyrest-e2e-tests/pull/3/files?diff=split&w=0#diff-7b9da19a85285c529a921548bf5bcada55174502a8d7c15beb7e6e75a8df12c9R16)

Reply:
This takes a relatively long time, but set this time in the config.

[e2e-tests/cypress.config.ts]
```typescript
env: {
  // Other variables
  readFileTimeout: 60000,
}
```
[e2e-tests/cypress/support/commands.ts]
```typescript
    const downloadTimeout = options.timeout || Cypress.env().readFileTimeout;
```